src/platform/web/ui/general/theming.js

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+export function getStyleSheetElements() {
```

Comment:
`general` is more for UI related DOM management code, perhaps put this in `src/platform/web/dom/theming.js`?

Reply:
This file got eliminated entirely now by introducing the `ThemeManager` class in response to your other comment above.